package.json

```diff
@@ -48,5 +48,8 @@
         "use-query-params": "^2.2.1",
         "zod": "^3.22.2",
         "zustand": "^4.4.0"
+    },
+    "resolutions": {
+        "@dhis2/ui": "8.14.2-alpha.1"
```

Comment:
Is alpha up2date with ui@master? I suppose that in that case this is fine, otherwise it might cause issues.

Reply:
We may want to wait until including this? Maybe in a separate PR?

Reply:
Yeah maybe you could extract the upgrade to ui@alpha and we handle adding the details panel separate from that (but we can keep the fixes for wrong prop usage here, would be a shame to have those)